tests/js/server/aql/aql-modify-noncluster.js

```diff
@@ -2187,10 +2217,12 @@ function ahuacatlInsertSuite () {
 
 function ahuacatlUpdateSuite () {
   var errors = internal.errors;
-  var cn1 = "UnitTestsAhuacatlUpdate1";
-  var cn2 = "UnitTestsAhuacatlUpdate2";
-  var c1;
-  var c2;
+  const cn1 = "UnitTestsAhuacatlUpdate1";
+  const cn2 = "UnitTestsAhuacatlUpdate2";
+  const cn3 = "UnitTestsAhuacatlUpdate3";
```

Comment:
Is it possible to go with one of the already existing collections here, to save test runtime?

Reply:
c1 and c2 were being used by other tests, so I added for separating the documents from my tests